input/contribute/developer-experience/the-pitch.md

```diff
@@ -7,4 +7,18 @@ Case in point - AngularJS was released, people adopted it, and then Version v2.0
 
 Internally the we debate whether ReactiveUI is or is not a framework, as at its core the project is essentially a bunch of extension methods for the Reactive Extensions.
 
-If you are sick of investing personal development energy in transitory frameworks whereby you must learn "how do I do $x with $framework specific way" knowing deep down in side that this knowledge has no value in the future; then please come and join us over at https://reactiveui.net/slack
\ No newline at end of file
+If you are sick of investing personal development energy in transitory frameworks whereby you must learn "how do I do $x with $framework specific way" knowing deep down in side that this knowledge has no value in the future; then please come and join us over at https://reactiveui.net/slack
+
+# Have you decided to adopt Rx?
+
+Have you solved a complex problem with Reactive Extensions in your code?  Did you implement a new feature using Rx and want to learn more about Reactive Programming?  Are you looking for a better way to deal with mutable state?  Have you heard of this horrible concept of Rx and want to know why its bad and you should stay away from it?  If the answer is yes to any of these questions, welcome to ReactiveUI.
+
```

Comment:
> Have you heard of this horrible concept of Rx and want to know why its bad and you should stay away from it
Needs to be a little more positive and inspiring - otherwise great. 

1. One of the areas we could talk about is observables return multiple values over time whereas tasks return only a single value. That's one of the core differences.
2. What about EventHandlers? Did you know _there is a better way?_  - > https://reactiveui.net/docs/handbook/events/